src/SDL_gpu_shadercross.c

```diff
@@ -560,6 +560,11 @@ static ID3DBlob *SDL_ShaderCross_INTERNAL_CompileDXBC(
     ID3DBlob *errorBlob;
     HRESULT ret;
 
+    if (SDL_D3DCompile == NULL) {
+        SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "%s", "Could not load D3DCompile!");
```

Comment:
Looks good to me, but perhaps we should make this library more SDL_mixer/SDL_image-alike?
There, the `IMG_Init` accepts flags that conditionally enables support for certain formats, and returns a mask of all actually enabled formats.
And perhaps this library should also make more use of `SDL_SetError`?

Reply:
Yeah. There's already a bunch of unused defines in there from when this was a header library and we need to take those out too.